Add Blue Birds with base BPM version, and ES JA

GuardedLolz opened the pull request:
### Prerequisites
* [X] I have read and understood the [contribution guidelines](CONTRIBUTING.md)
* [X] This pull request follows the code style of the project
* [ ] I have tested this feature thoroughly WITH THE USE of the load-all-sounds starter
  * [Link to a video of proof of testing]()

### Changes Proposed in this Pull Request
* Add pitch-changing versions of the Blue Birds cues
* Add Spanish and Japanese Blue Birds

...

GuardedLolz committed:
Save Base BPM!

GuardedLolz committed:
Add stuff to games.json

chrislo27 commented on core/assets/data/games.json:
```diff
@@ -52,8 +53,8 @@
   "lumbearjack",
   "manzaiBirds",
-  "marchingOrders",
   "marchingOrdersEn",
```
Retain alphabetical order.

chrislo27 commented on core/assets/data/games.json:
```diff
@@ -88,8 +89,8 @@
   "sickBeats",
   "sneakySpirits",
-  "spaceDance",
   "spaceDanceEn",
```
Retain alphabetical order

chrislo27 reviewed (commented):
Looks fine with a quick skim. Will wait for the video.

GuardedLolz committed:
Go logic.

GuardedLolz commented:
Then for the sake of consistency, I will change the orders like this:
Japanese, then English, then Spanish.

chrislo27 commented:
Use alphabetical order.

GuardedLolz committed:
Fine.

I just did it. I DID SOMETHING COMPLETELY COSMETIC JUST TO PLEASE
YOU!!!!!!!!!
A R E Y O U H A P P Y N O W ?

GuardedLolz commented:
Sorry about my bad mood. It's just the inconvenient situation.

GuardedLolz commented:
I'm sorry. I don't mean to be rude.

chrislo27 commented:
Nah, you weren't rude. I have a tendency to be straightforward when reviewing stuff (Discord4J taught me how) :P

chrislo27 commented on core/assets/sounds/cues/spaceDanceJa/data.json:
```diff
@@ -1,184 +1,204 @@
     {
-      "id": "spaceDance_pupupupunch1",
+      "id": "spaceDanceJa_pupupupunch1",
+      "deprecatedIDs": ["spaceDance_pupupupunch1"]
```
Missing comma. It's evident that you didn't test the PR with all sounds loaded because this error would've been caught.

chrislo27 commented on core/assets/sounds/cues/spaceDanceJa/data.json:
```diff
@@ -1,184 +1,204 @@
     {
-      "id": "spaceDance_turnright2",
+      "id": "spaceDanceJa_turnright2",
+      "deprecatedIDs": ["spaceDance_turnright2"]
```
Missing comma.

chrislo27 commented on core/assets/sounds/cues/spaceDanceJa/data.json:
```diff
@@ -1,184 +1,204 @@
     {
-      "id": "spaceDance_letssitdown2",
+      "id": "spaceDanceJa_letssitdown2",
+      "deprecatedIDs": ["spaceDance_letssitdown2"]
```
Missing comma.

chrislo27 commented on core/assets/sounds/cues/spaceDanceJa/data.json:
```diff
@@ -1,184 +1,204 @@
     {
-      "id": "spaceDance_pupupupunch2",
+      "id": "spaceDanceJa_pupupupunch2",
+      "deprecatedIDs": ["spaceDance_pupupupunch2"]
```
Missing comma.

chrislo27 reviewed (changes requested):
It is obvious you do not test your changes (otherwise these syntax errors would be gone!). Because you are deprecating some IDs, please test saving all the deprecated IDs and then loading it with your version for it to make sure it properly converts.

Syntax errors: http://i.imgur.com/EwllgZv.png

chrislo27 commented:
I ran a script to check deprecations: all good.

GuardedLolz commented:
I am going to close the pull request here and do a bigger one someday.

chrislo27 commented:
:thinking: It would be fine if you had just fixed those four syntax errors (+ tested it).
